pandas/core/interchange/column.py

```diff
@@ -269,7 +274,11 @@ def _get_data_buffer(
             DtypeKind.BOOL,
             DtypeKind.DATETIME,
         ):
-            buffer = PandasBuffer(self._col.to_numpy(), allow_copy=self._allow_copy)
+            if self.dtype[0] == DtypeKind.DATETIME and len(self.dtype[2]) > 4:
```

Comment:
so anything longer than `'tsn:'` will go here (like `tss:US/Pacific`), nice

Reply:
Yeah correct. Added a comment to that effect